pkg/genericapiserver/genericapiserver.go

```diff
@@ -892,6 +894,25 @@ func (s *GenericAPIServer) InstallSwaggerAPI() {
 		},
 	}
 	swagger.RegisterSwaggerService(swaggerConfig, s.HandlerContainer)
+	openAPIConfig := openapi.Config{
```

Comment:
What is the difference between this and swagger 1.2? Both of them (together) are optional and controlled by a single flag Config.EnableSwaggerSupport that I think make sense.